RecoVertex/PrimaryVertexProducer_Alpaka/test/compare.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+cmsRun ../../PrimaryVertexProducer/test/testPrimaryVertexProducer_CPU.py
+
+cmsRun testPrimaryVertexProducer_Alpaka.py --backend $i
+
+python compareAlgos.py testAlpaka.root testCPU.root
```

Comment:
This script alone seems to not to be very useful. The files
* `../PrimaryVertexProducer/test/testPrimaryVertexProducer_CPU.py`
* `testPrimaryVertexProducer_Alpaka.py`
* `compareAlgos.py`

Are these files perhaps just missing from the PR?